foss4gna_24/big-EO-viz.ipynb

```diff
@@ -0,0 +1,1460 @@
+{
```

Comment:
<p>there is a Python way to set the env variable, but that can be done later</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/geotrees-demo/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
updated with - 

```import boto3
from botocore.config import Config

# Set the AWS region
REGION = "us-west-2"

# Configure a boto3 session with the specified region
session = boto3.Session(region_name=REGION)
```